package/MDAnalysis/topology/ParmEdParser.py

```diff
@@ -0,0 +1,317 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+# doi: 10.25080/majora-629e541a-00e
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+"""
+ParmEd topology parser
+===================
+
+Converts a ParmEd_ structure into a Topology.
+
+.. _ParmEd: https://parmed.github.io/ParmEd/html/index.html
+
+Example
```

Comment:
I *think* such a heading will not render nicely with numpy docs/napoleon. We normally rewrite it to not be the word "Example" or "Examples", such as "Example use" or something more meaningful.

(I thought we had a note regarding this problem at https://github.com/MDAnalysis/mdanalysis/wiki/WritingDocumentation#numpy-style-gotchas but we don't so I am not 100% sure if this is still a problem.)

Reply:
I think normally the heading used in MDAnalysis is just Example/s? It displays alright for [Universe](https://www.mdanalysis.org/mdanalysis/documentation_pages/core/universe.html#MDAnalysis.core.universe.Universe.add_TopologyAttr).